src/main/java/hinlok/Hinlok.java

```diff
@@ -0,0 +1,42 @@
+package hinlok;
+
+import hinlok.command.Command;
+import hinlok.parser.Parser;
+import hinlok.tasks.TaskList;
+import hinlok.file.TaskFile;
+import hinlok.ui.Ui;
+
```

Comment:
Do you think writing javadocs will make the class and methods' purpose more clear?

Reply:
yes, will do!